mapper-processor/src/main/java/com/datastax/oss/quarkus/runtime/internal/mapper/processor/QuarkusCodeGeneratorFactory.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.internal.mapper.processor;
```

Comment:
Package names in this module are a bit funky. If you agree to split this in two modules, then we would have:

1. `cassandra-quarkus-mapper-runtime`: `com.datastax.oss.quarkus.internal.mapper.runtime`
2. `cassandra-quarkus-mapper-processor`: `com.datastax.oss.quarkus.internal.mapper.processor`